mod/display.php

```diff
@@ -11,6 +11,17 @@ function display_init(App $a) {
 	$nick = (($a->argc > 1) ? $a->argv[1] : '');
 	$profiledata = array();
 
+	if ($a->argc == 3) {
+		if (substr($a->argv[2], -5) == '.atom') {
+			require_once('include/dfrn.php');
```

Comment:
Do we have a general rule for that? This is some special case, mostly we don't need that include.

I was unsure where to place it, but of course I can place it on the top.